wdae/wdae/helpers/dae_query.py

```diff
@@ -4,10 +4,6 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def columns_to_labels(columns, column_labels):
-    return [column_labels[column] for column in columns]
-
-
 def join_line(l, sep='\t'):
```

Comment:
consider moving this function to StudyWrapper

Reply:
It is moved to dae_utils in dae/utils and downloading of 
    * Genotype Browser
    * Common Report families data
    * Gene Weights
is moved in dae module.